sparkle/platform/output.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
+"""Sparkle class to organise output."""
+
+from __future__ import annotations
+
+from CLI.help import global_variables as gv
+# from sparkle.platform import generate_report_for_selection as sgfs
```

Comment:
These report generation imports should be removed (Maybe you just did this for testing, but just as a reminder)

Reply:
I'll need this when working on the SelectionOutput class, if I remember correctly the class has the calculate_marginal_contribution function

Reply:
Yes but this should not be called in this file, rather the values should be handed to it in the constructor

Reply:
Do note that the selection report is the worst mess of all three, making everything quite confusing. I'll see what I can do refactoring wise